.github/workflows/ci.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19.12
+          go-version: 1.21.4
```

Comment:
is this a must? if we want to upgrade go version, should do in another PR

Reply:
Yes, go-ethereum has been upgraded to go1.21, we must upgrade too